raiden/network/pathfinding.py

```diff
@@ -18,6 +20,36 @@
 log = structlog.get_logger(__name__)
 
 
+@unique
+class PfsError(IntEnum):
```

Comment:
Nitpick ... but I feel strongly about it. Please rename this to `PFSError`. You use it capitalized as `PFS` in other places already and well, it's already an abbreviation so should be capitalized.

Reply:
Right, consintent naming is important.